07-html.Rmd

```diff
@@ -408,3 +408,60 @@ In Section \@ref(latex-hardcore), we mentioned that if you feel the constraint o
+
+To add a favicon to your HTML document, add the following line of code to a custom header file (such as the file `header.html` mentioned in \@ref(include-html)):
+
+```html
+<link rel="shortcut icon" href="{path to favicon file}">
+```
+
+Recall that this file can be injected into the document `<head>` area using the YAML metadata.
+
+```yaml
+output:
+  html_document:
+    includes:
+      in_header: header.html
+```
+
+The path you provide to `href` should assume the same relative path structure as you would use to reference any other asset (e.g. an image or data set). For the image itself, most small, square PNG files will work reasonably well Bear in mind that a typical web browser will often display the image in a 16 x 16 pixel box, so simple designs are better. 
+
+If you want to ensure that each browser or platform on which your document is viewed uses a version of your icon with optimal resolution for its specific layout, you may use a service such as https://realfavicongenerator.net/ to generate a set of favicons and a slightly more complex version of the header HTML code. This service is currently used by the **pkgdown** package's `pkgdown::build_favicon()` function to make a set of favicons out of R package logos. 
+
+
+## Sharing HTML content on the web
+
+One appealing aspect of rendering R Markdown content to HTML files is that it is very easy to host these files on the internet and share them just as one shares any other website. This section briefly summarizes numerous options for sharing the HTML documents which you create. 
+
+### R-specific services
+
+RStudio PBC offers a number of services for publishing various types of content created in R Markdown to the internet. These services make it particularly easy to publish content by using the RStudio IDE or the **rsconnect** package.
+
+- [**RPubs**](https://rpubs.com/) enables free hosting of static single-file R Markdown content. It is easy to publish via the Publish button in the RStudio IDE or the `rsconnect::rpubsUpload()` function. Please see the "Getting Started" page (https://rpubs.com/about/getting-started) for more details.
+
+- [**shinyapps.io**](https://www.shinyapps.io/) allows for hosting dynamic content which requires a server running R. For example, one can host interactive R Markdown documents which use the `runtime: shiny` option to include Shiny components. shinyapps.io is an analog to RPubs for Shiny applications. Apps and interactive R Markdowns can be published using the push-button in the RStudio IDE or the `rsconnect::deployApp()` function. See the user guide (https://docs.rstudio.com/shinyapps.io/index.html) for more details. 
+
+- [**bookdown.org**](https://bookdown.org/home/about/) offers free hosting specifically for books written with the **bookdown** package. You may easily publish static output files of your book using the `bookdown::publish_book()` function. This platform is based on RStudio PBC's enterprise RStudio Connect product (see below).
```

Comment:
> This platform is based on RStudio PBC's enterprise RStudio Connect product (see below).

not sure this is worth mentioning. I see bookdown.org as a service, and you do not need to know how it works behind. 😄  like shinyapps.io 😉

Reply:
Good point. In retrospect, this was a very arbitrary detail to include. Removed!